docs/api/proposed_endpoints.md

```diff
@@ -402,17 +405,35 @@ Status: 201 Created
 ## Update proposal details
 
 ```
-PATCH /api/proposal/:id (request authentication)
+PATCH /api/proposals/:id (requires authentication)
 ```
 
 **Request**
 ```json
 {
-    "title": "Corrected title of talk",
-    "level": "advanced"
+    "id": "0f342ac1-ac32-4bd1-3612-efa32bc3d9a0",
```

Comment:
Since it's a `PATCH` request and we are providing `id` in the URL itself, no need to keep it in the request then.